app/pb_kits/playbook/pb_inventory_location/_inventory_location.jsx

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/*eslint-disable react/no-multi-comp, flowtype/space-before-type-colon */
+
+import React from 'react'
+import LabelValue from '../pb_label_value/_label_value.jsx';
+import IconBodyTitle from '../pb_icon_body_title/_icon_body_title.jsx';
+
+type InventoryLocationProps = {
+  bin?: String,
```

Comment:
By default, our kit generator is adding the optional `?` to these prop names. We might actually turn this off in the future, but anything that doesn't have a default value or guarding logic should probably not have these.